coala_quickstart/info_extractors/PackageJSONInfoExtractor.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from coala_quickstart.info_extraction.InfoExtractor import InfoExtractor
+from coala_quickstart.info_extraction.Information import (
+    LicenseUsedInfo, ProjectDependencyInfo, IncludePathsInfo, ManPathsInfo,
+    VersionInfo)
+
+
+class PackageJSONInfoExtractor(InfoExtractor):
+    supported_file_globs = ("package.json",)
```

Comment:
quick question: the `.` in the glob isn't "match any character", right? Do we need to escape it? I don't quite remember how our globbing works :D 

The simplest way to test would be to create a file called "packagexjson" and see if it's picked up.